connect/src/main/java/com/stripe/android/connect/webview/StripeConnectWebViewContainer.kt

```diff
@@ -183,6 +186,7 @@ internal class StripeConnectWebViewContainerImpl<Listener : StripeEmbeddedCompon
         val viewBinding = this.viewBinding ?: return
         viewBinding.stripeWebView.setBackgroundColor(state.backgroundColor)
         viewBinding.stripeWebViewProgressBar.isVisible = state.isNativeLoadingIndicatorVisible
+        viewBinding.stripeWebView.isVisible = !state.isNativeLoadingIndicatorVisible
```

Comment:
👀 if there's an error, it's a better user experience to hide the webview (which might have weird rendering issues) and show the spinner. We might consider an explicit error screen, but it would be difficult to determine when to show it.